package.json

```diff
@@ -595,6 +595,7 @@
         "build-watch": "npm run build -- --watch"
     },
     "dependencies": {
+        "@stylistic/eslint-plugin-ts": "^2.6.4",
```

Comment:
Please follow the example set by the other deps (no ^ modifier).

Reply:
not all of the other deps are without "^"

this was added by the npm tool itself

`npm install @stylistic/eslint-plugin-ts`

Reply:
https://github.com/mike-lischke/vscode-antlr4/blob/master/package.json

there is no consisentency regarding "^", so I just trusted the npm tool in this instance

Reply:
Yes, npm always adds the "^" modifier, still I don't want them. Sometimes I miss an entry, so it still has that and needs to be fixed.